src/styles.scss

```diff
@@ -1999,10 +1999,20 @@ input {
   background-color: $profile-button;
 }
 .btn-primary {
-  background-color: #005BFF;
-  border-color: #005BFF;
+  background-color: $buy-button-blue;
+  border-color: $buy-button-blue;
   color: #fff;
 }
+.btn-primary.disabled, .btn-primary:disabled {
+	color: #fff;
+	background-color: $buy-button-blue;
+	border-color: $buy-button-blue;
+}
+.btn-primary:hover {
```

Comment:
for some reason, when you hover over a disabled button, it changes color, which it shouldn't do when disabled.

Reply:
Ok got it cause hover is after disabled class